src/metabase/sync/sync_metadata.clj

```diff
@@ -50,7 +51,8 @@
    ;; Now for each table, sync the FKS. This has to be done after syncing all the fields to make sure target fields exist
    (sync-util/create-sync-step "sync-fks" sync-fks/sync-fks! sync-fks-summary)
    ;; finally, sync the metadata metadata table if it exists.
```

Comment:
this comment needs to update

Reply:
Updated 78bc720b69a7b254b75f1390f215a35f975dee83